math/finding_number_of_digits_in_a_number.cpp

```diff
@@ -0,0 +1,33 @@
+/**
+ * @author amino19
+ * @file
+ *
+ * \brief Finding number of Digits in a Number
+ * \details Basically, its a very basic math of
```

Comment:
You should provide a better-detailed description (see, for example, the [`n_queens.cpp`](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/backtracking/n_queens.cpp) algorithm).

Reply:
Actually its a very short code, but I'll try my best to figure it out!